### PR TITLE
Changelog for Clippy 1.82 :airplane: 

### PR DESCRIPTION
**Pull request by xFrednet**
```
Roses are red,
Violets are blue,
EuroRust in Austria,
    RustConf in Canada.
```

---

### The cat of this release is *Racka*:

The cats of this Clippy release

Cats for the next release can be nominated in the comments :D

---

changelog: none

### CODE REVIEW

File: changes.md
Comment (flip1995) [inline changes.md:1]: I think this file was committed by accident

File: clippy_lints/src/unnecessary_struct_initialization.rs [hunk 1]
```diff
@@ -38,7 +38,7 @@ declare_clippy_lint! {
     /// moved out of, see [#10547](https://github.com/rust-lang/rust-clippy/issues/10547).
     ///
     /// Empty structs are ignored by the lint.
-    #[clippy::version = "1.70.0"]
+    #[clippy::version = "1.82.0"]
```
Comment (y21) [inline clippy_lints/src/unnecessary_struct_initialization.rs:41]: The version was correct I think. The lint has definitely existed for longer than this version at least. It was only extended to catch more patterns recently

Comment (xFrednet) [reply clippy_lints/src/unnecessary_struct_initialization.rs]: Right, there was something confusing where a changelog entry said it added a lint and that was the only one with changes. On the plain I didn't have WIFI to check the PR ^^

Should be correct now

**Comment by Centri3** [comment]
r? rust-lang/clippy

**Review by dswij — APPROVED** [review]
Thanks for preparing the changelog!

**Comment by jdonszelmann** [comment]
IMG_20220611_142815-1

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,11 +6,46 @@ document.
...
+
+## Rust 1.82
+
+Current stable, released 2024-10-17
+
+[View all 108 merged pull requests](https://github.com/rust-lang/rust-clippy/pulls?q=merged%3A2024-07-11T20%3A12%3A07Z..2024-08-24T20%3A55%3A35Z+base%3Amaster)
+
+### New Lints
+
+* Added [`too_long_first_doc_paragraph`] to `style`
```
Comment (xFrednet) [inline CHANGELOG.md:19]: Update this if we move the lint to nursery: https://rust-lang.zulipchat.com/#narrow/stream/257328-clippy/topic/too_long_first_doc_paragraph.20to.20nursery.3F/near/476448239


```suggestion
* Added [`too_long_first_doc_paragraph`] to `nursery`
```

**Comment by cuviper** [comment]
> ```
> Roses are red,
> Violets are blue,
> EuroRust in Austria,
>     RustConf in USA.
> ```

Maybe I'm missing a joke, but RustConf was in Canada this year.

**Comment by xFrednet** [comment]
> Maybe I'm missing a joke, but RustConf was in Canada this year.

Woops, that was an unintentional error, thanks for pointing it out! I just misremembered and USA fit with the formatting. It's fixed now

**Comment by and-reas-se** [comment]
I would like to nominate Vera.

vera-2024

**Comment by xFrednet** [comment]
Absolutely beautiful cats :3

We should probably merge this today

---

Cats are cute,
Submissions being reviewed,
This has been approved,
Let's get this moved.

**Pull request merged**